src/test/resources/features/convertExtractText/ExtractText.feature

```diff
@@ -0,0 +1,80 @@
+    When I send a POST request to "/convertExtractText" with the following form data
+      | file         | @"template/img/demo.png"         |
+      | md5          | f31e75933983501423ca55e176ca163e |
+      | language     | spañol                           |
+      | exportFormat | .txt                             |
+    Then I validate the response has the "400" status code
+    And I validate that the response body should match with "common/errorResponse.json" JSON schema
+    And I validate that the response contain the following values
+      | status  | 400                      |
+      | error   | "Error invalid language" |
+
+  @negative
+  Scenario: Verify that is not possible to extract text with an invalid md5
+    When I send a POST request to "/convertExtractText" with the following form data
+      | file         | @"template/img/demo.png"         |
+      | md5          | f31e75933983501423ca55e176ca163w |
+      | language     | spa                              |
+      | exportFormat | .txt                             |
+    Then I validate the response has the "400" status code
+    And I validate that the response body should match with "common/errorResponse.json" JSON schema
+    And I validate that the response contain the following values
+      | status  | 400                 |
+      | error   | "Failed in the md5" |
+
+  @negative
+  Scenario: Verify that is not possible to extract text with an empty language
+    When I send a POST request to "/convertExtractText" with the following form data
+      | file         | @"template/img/demo.png"         |
+      | md5          | f31e75933983501423ca55e176ca163e |
+      | language     |                                  |
+      | exportFormat | .txt                             |
+    Then I validate the response has the "400" status code
+    And I validate that the response body should match with "common/errorResponse.json" JSON schema
+    And I validate that the response contain the following values
+      | status | 400                     |
+      | error  | "failed Language empty" |
+
+  @negative
+  Scenario: Verify that is not possible to extract text with an empty exportFormat
+    When I send a POST request to "/convertExtractText" with the following form data
+      | file         | @"template/img/demo.png"         |
+      | md5          | f31e75933983501423ca55e176ca163e |
+      | language     | spa                              |
+      | exportFormat |                                  |
+    Then I validate the response has the "400" status code
+    And I validate that the response body should match with "common/errorResponse.json" JSON schema
+    And I validate that the response contain the following values
+      | status | 400                     |
+      | error  | "Failed format empty" |
+
+  @negative
+  Scenario: Verify that is not possible to extract text with empty form data
```

Comment:
Please consider changing this title to something like: `Verify that is not possible to extract text without configuration parameters`